joomla-projects/soc21_website-cronjob
# Add webcron to scheduler config

### Summary of Changes
Auto-generate the webcron key and enhance scheduler config form.


### Testing Instructions
Test the scheduler component configuration for the webcron key. It should auto-generate on enabling webcron and saving the form. Toggling the reset radio should reset the key on form save.
Webcron base link should be copy-able through the link field in the form, which is visible when the webcron feature is enabled and a key has been generated.
